packages/core/package.json

```diff
@@ -8,7 +8,8 @@
         "pre": "node scripts/prepare",
         "build": "tsc",
         "test-cli": "./scripts/testCLI.sh",
-        "test": "yarn synchronizerTests && yarn UnirepStateTests & yarn UserStateTests & yarn integrationTest",
+        "test": "yarn synchronizerTests && yarn UnirepStateTests && yarn UserStateTests && yarn integrationTest",
+        "test-ci": "hardhat test --parallel",
```

Comment:
do we need `test-ci` here?
I think we will change `test` in https://github.com/Unirep/Unirep/pull/58 so maybe we can remove `test-ci`